webapp/i18n/en.json

```diff
@@ -240,7 +240,8 @@
   "ViewTitle.show-description": "show description",
   "ViewTitle.untitled-board": "Untitled board",
   "WelcomePage.Description": "Boards is a project management tool that helps define, organize, track and manage work across teams, using a familiar kanban board view",
-  "WelcomePage.Explore.Button": "Explore",
+  "WelcomePage.Explore.Button": "Take a tour",
```

Comment:
Unrelated?

Reply:
Nope, totally related. We're changing the text of buttons that trigger the change in user config.